beacon/reactor.go

```diff
@@ -391,7 +390,6 @@ func (ps *PeerState) setLastComputedEntropyHeight(height int64) {
 
 func (ps *PeerState) sendEntropy(nextEntropyHeight int64, entropy types.ThresholdSignature) {
 	// Send peer entropy from block store
-	ps.logger.Debug("sendEntropy", "ps", ps, "height", nextEntropyHeight)
```

Comment:
did you want to remove this debug?

Reply:
Yeah, these debug messages are too much and not very helpful. I found it just really obscured the logs.